frontend/src/app/menu/menu.component.spec.ts

```diff
@@ -1,18 +1,22 @@
-import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
-import { NoopAnimationsModule } from '@angular/platform-browser/animations';
-import { RouterTestingModule } from '@angular/router/testing';
-import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { waitForAsync, ComponentFixture, TestBed } from "@angular/core/testing";
+import { NoopAnimationsModule } from "@angular/platform-browser/animations";
+import { HttpClientTestingModule } from "@angular/common/http/testing";
 
-import { MenuComponent } from './menu.component';
-
-describe('MenuComponent', () => {
+import { MenuComponent } from "./menu.component";
+import { RouterModule } from "@angular/router";
+import { routes } from "../routes";
+describe("MenuComponent", () => {
     let component: MenuComponent;
     let fixture: ComponentFixture<MenuComponent>;
 
     beforeEach(waitForAsync(() => {
         TestBed.configureTestingModule({
             declarations: [MenuComponent],
-            imports: [NoopAnimationsModule, RouterTestingModule, HttpClientTestingModule]
```

Comment:
Apparently, the RouterTestingModule is deprecated. The regular RouterModule should take care of everything.